apps/web/src/lib/uploadToIPFS.ts

```diff
@@ -49,7 +49,8 @@ const uploadToIPFS = async (data: any): Promise<LensterAttachment[]> => {
     );
 
     return attachments;
-  } catch {
+  } catch (error) {
+    console.error(error);
```

Comment:
we don't want to throw errors on prod!! maybe in future we can make a logger lib that yells only on development

Reply:
I assume you mean we should remove the log here. I'll remove it! I think your suggestion about a logger library is important to follow up on. It can save many hours of troubleshooting if there actually is a bug.

My reasoning behind keeping the log output was that I think that in general, a catch clause should always be handled, even if it's only with log, be it on client or server side.